engine/src/main/scala/cromwell/engine/workflow/lifecycle/materialization/MaterializeWorkflowDescriptorActor.scala

```diff
@@ -275,6 +275,7 @@ class MaterializeWorkflowDescriptorActor(serviceRegistryActor: ActorRef,
       factory <- findFactory(sourceAndResolvers._1).toIOChecked
       outputRuntimeExtractor <- factory.womOutputRuntimeExtractor.toValidated.toIOChecked
       validatedNamespace <- buildValidatedNamespace(factory, sourceAndResolvers._1, sourceAndResolvers._2)
+      _ = sourceAndResolvers._2.foreach(_.close())
```

Comment:
This means that any failure in `close()` will  be ignored. Which might be what you intended, but in that case it feels awkward that we're returning `Try`s from them in any case.

As an exercise in "wouldn't it be nice"ing, perhaps `close` could return an ErrorOr and we could traverse the `close()`s here to get a single "did anything fail" result. And then separately we could decide whether to log, ignore, or fail the workflow as a result?

Reply:
I chose `Try` somewhat arbitrarily and fully support adopting the latest and greatest `ErrorOr`

That said there's nothing we can do about a failure ** for this particular case **

Reply:
I would think a log message at least is appropriate?

Reply:
Agreed, I missed that suggestion